R/read.R

```diff
@@ -1,11 +1,12 @@
 orderly_read <- function(path, call = NULL) {
-  assert_file_exists_relative("orderly.R", name = "Orderly file",
+  orderly_name <- deprecate_old_orderly_name(path, basename(path))
+  assert_file_exists_relative(orderly_name, name = "Orderly file",
                               workdir = path, call = call)
```

Comment:
Is this line necessary? Doesn't `deprecate_old_orderly_name` always return a path to an existing file?

Reply:
no the assert file exists check is not necessary anymore!